### ISSUE TITLE
Repeated calls to `.type()` result in unexpected behavior

### ISSUE THREAD

**Issue by agj** [alexmacarthur/typeit#121]
Thanks for this library. I'm trying to use it interactively, but running into issues. [Here's a codepen](https://codepen.io/anon/pen/KBMXXm?editors=1010) demonstrating the problem, with version 5.10.1. The code used is below:

```html
<script src="https://cdn.jsdelivr.net/npm/typeit@5.10.1/dist/typeit.min.js"></script>

<div>
	<div id="text"></div>
	<div id="next" style="width: 100px; height: 100px; background: red">Next</div>
</div>
```

```js
const onLoad = cb => /interactive|complete/.test(document.readyState) ? setTimeout(cb, 0) : document.addEventListener('DOMContentLoaded', cb);

onLoad(() => {
	const texts = [
		"Something to type first.",
		"Something to type next." ,
		"And something to type last."
	];
	let currentText = 0;
	const typeIt = new TypeIt('#text');

	document.querySelector('#next')
	.addEventListener('click', () => {
		typeIt.type(texts[currentText]);
		currentText += 1;
	});
});
```

Try pressing the red button that says 'Next' three times in succession. It will fire `.type()` with different strings each time, but resulting in gibberish typed out, like this:

```
Somethgint o type firstS.motehigno t epyt ne.xtAdn somheting to yetaps l.t
```

**Comment by alexmacarthur** [comment]
Thanks for reporting this! digging in.

**Comment by alexmacarthur** [comment]
I just released version 5.10.2, which resolves this issue. Try plugging that into your CodePen and you'll notice there's no more jumbled text. 

Thanks again for reporting this issue!

**Comment by agj** [comment]
I just realized I never replied thanking you for this. Truly sorry! So, thanks a bunch for the fix! :)
